File: Gombang/Weekly Contest 400/3168_Minimum_Number_of_Chairs_in_a_Waiting_Room.cs
public class Solution
{
    public int MinimumChairs(string s)
    {
        int currentChairs = 0;

        int result = 0;
        for (int i = 0; i < s.Length; i++)
        {
            if (s[i] == 'E')
            {
                currentChairs++;
                if (result < currentChairs)
                    result = currentChairs;
            }
            else
                currentChairs--;
        }

        return result;
    }
}

File: Seori/Weekly Contest 400/3168. Minimum Number of Chairs in a Waiting Room.py
class Solution:
    def minimumChairs(self, s: str) -> int:
        answer = 0
        chairs = 0
        for letter in s:
            # [1] If the letter is 'E', increment the number of chairs
            if letter == 'E':
                chairs += 1
            # [2] If the letter is 'L', check maximum chairs and decrement the number of chairs
            elif letter == 'L':
                answer = max(answer, chairs)
                chairs -= 1

        answer = max(answer, chairs)
        return answer